justifi-tech/web-component-library
# Build and workspace cleanup in preparation for Lerna publishing

- New node version (`16.17.0` --> `18.17.1`) required by `.nvmrc`, and `.nvmrc` moved to root directory so that everything is using the same node version
- New `@stencil/core` version (`3.3.0` --> `4.0.5`) - reviewed breaking changes reported by stenciljs docs, seems as though we were not using any of the features expected to break. Project builds and runs without any noticeable issues.
- Removal of `release-it` and `autochangelog` files and `package.json` entries as they are no longer used
- Use `npm` instead of `yarn` to simplify / reduce factors at play with this PR
- Updated the `README` to note use of Lerna and Lerna commands, could use more work in the near future to clean up and documentation of commands, running and testing the project, and to consolidate all commands at the root level to make the developer experience as easy as possible


Steps for Testing/QA
--------------------
- Clone this repo to a new place on your machine such as the desktop, so that you have a brand new copy of the project
- Within the newly cloned repo:
  - run `npm install`
  - run `npm run build`
  - run `npm run test`
  - change directory into `stencil-library` and run `npm run start`. The project should start up without issue. Within the running storybook instance, check each component to see that it is rendering properly
